pkg/sandbox/runx/impl/runx.go

```diff
@@ -0,0 +1,13 @@
+package impl
+
+import "go.jetpack.io/pkg/sandbox/runx/impl/runopt"
+
+type RunX struct {
+	GithubAPIToken string
+}
```

Comment:
My preference would be to put this struct directly in the `runx` package and remove the interface. Because the GitHub API token is optional, I'd also remove `runopt` in favor of setting the `GithubAPIToken` field directly. For example:

```go
var r runx.Runx
// -- or --
// r := runx.RunX {
//     GithubAPIToken: "token",
// }
r.Run(ctx, "golangci-lint", "run")
```